.github/workflows/check-semver.yml

```diff
@@ -22,24 +27,46 @@ jobs:
       - name: Rust compilation prerequisites
         run: |
           rustup default nightly-2024-03-01
-          rustup target add wasm32-unknown-unknown --toolchain nightly-2024-03-01
           rustup component add rust-src --toolchain nightly-2024-03-01
 
       - name: install parity-publish
-        run: cargo install parity-publish@0.5.1
+        run: CARGO_TARGET_DIR=./target/ cargo install parity-publish -q --git https://github.com/paritytech/parity-publish --branch oty-max-allowed-bump
 
       - name: extra git setup
         run: |
           git config --global --add safe.directory '*'
           git fetch --no-tags --no-recurse-submodules --depth=1 origin master
           git branch old origin/master
 
+      - name: Comment If Backport
+        if: github.event.pull_request.base.ref == 'stable'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR: ${{ github.event.pull_request.number }}
+        run: |
+          echo "This is a backport into stable."
+
+          wget -q https://github.com/cli/cli/releases/download/v2.51.0/gh_2.51.0_linux_amd64.tar.gz -O gh.tar.gz && \
+          tar -xzf gh.tar.gz && mv gh_2.51.0_linux_amd64/bin/gh /usr/local/bin/gh && rm gh.tar.gz
+          chmod +x /usr/local/bin/gh
```

Comment:
https://github.com/actions/github-script#comment-on-an-issue maybe we could use this?

Reply:
I tried it, but this and similar actions dont allow to simply edit the last commit. The gh cli has a `--edit-last` flag to avoid spam in the merge request that just amends it.  
Alternative is to use the gh API and figure out the last comment id, and then editing that id. Surely more difficult than this one command.